### ISSUE TITLE
Line endings

### ISSUE THREAD

**Issue by Perlence** [pocke/lemonade#15]
Should lemonade convert line endings? For example if you copy something on Windows host and `lemonade paste` it on Linux host, there are `\r` characters that are almost never needed.

**Comment by pocke** [comment]
It's a nice idea.
However, I don't think lemonade should convert it as default. When copy a binary file, lemonade may malfunction.

I'll add a option for converting line ending.


example
------



`lemonade.toml` for lemonade server. https://github.com/pocke/lemonade#server

```toml
port = 1234
allow = '192.168.0.0/24'
convert_eol = true
```

**Comment by pocke** [comment]
@Perlence I've implemented this feature at #16 .

https://github.com/pocke/lemonade/releases/tag/v1.1.0



Please try the following option.
https://github.com/pocke/lemonade#line-ending

```sh
# on linux
$ lemonade paste --line-ending lf  # => converted text
```

or add `line-ending` section into `lemonade.toml` on linux
https://github.com/pocke/lemonade#client

```toml
line-ending = "lf"
```

**Comment by Perlence** [comment]
Works like a charm, thank you :+1:
